index.html

```diff
@@ -270,13 +408,21 @@ <h2 class="about-me-header">Software Developer</h2>
         </ul>
       </section>
 
-      <div class="social-media-div">
-        <ul class="social-media-ul display-flex">
+      <div
+        class="social-media-div display-flex-lg margin-0auto-lg justify-content-space-between"
+      >
+        <div class="display-flex justify-content-center">
+          <img
+            src="./images/arrow-down.png"
+            class="arrow-down"
+            alt="arrow down"
+          />
```

Comment:
- [ ] Please, consider that icon for scroll down should be one in mobile version. You have it doubled. I think it comes from the desktop version, where it should be doubled. 

<img width="371" alt="Screen Shot 2022-02-17 at 14 33 58" src="https://user-images.githubusercontent.com/49064106/154458178-44ba3768-f1b8-444f-8f3b-c0e94ac14976.png">

Reply:
Thanks, @elmar8287  changes made